coremltools/models/model.py

```diff
@@ -485,40 +485,55 @@ def predict(self, data):
 
         Parameters
         ----------
-        data: dict[str, value]
-            Dictionary of data to make predictions from where the keys are
-            the names of the input features.
-            If value is array type, numpy.ndarray, tensorflow.Tensor and torch.Tensor are acceptable.
```

Comment:
Do we really support tensor flow.Tensor or torch.Tensor as values? I thought only numpy arrays or PIL images are supported.

Reply:
We do support those input types. [It was a community contribution](https://github.com/apple/coremltools/pull/1459).